test/server/cards/02_SHD/units/DoctorPershingExperimentingWithLife.spec.ts

```diff
@@ -0,0 +1,86 @@
+describe('Doctor Pershing, Experimenting With Life', function() {
+    integration(function(contextRef) {
+        describe('Doctor Pershing\'s action ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['doctor-pershing#experimenting-with-life', 'wampa'],
+                        spaceArena: ['tieln-fighter']
+                    },
+                    player2: {
+                        groundArena: ['frontier-atrt'],
+                        spaceArena: ['cartel-spacer']
+                    }
+                });
+            });
+
+            it('should deal 1 damage to any selected friendly unit and draw a card', function () {
```

Comment:
Since this is our first instance of damage as a cost, do you mind also adding an integration test case for me? Specifically I want to confirm that if he does his damage to a unit with a shield that it still satisfies the condition. We haven't had a lot of opportunities to confirm that replacement effects work correctly for paying costs yet.

If it has any issues you can just reach out to me or start a different PR if you're interested to do it yourself (either is fine). It's too big an issue to be worth including in this PR if there's a problem there.

Reply:
I've added a new test scenario to check this!